app/models/case_log.rb

```diff
@@ -247,6 +251,10 @@ def mandatory_fields
       dynamically_not_required << "ecstat#{n}"
     end
 
+    if net_income_known != "Prefer not to say"
```

Comment:
Think we also need `No` here?

Reply:
I wouldn't think that income not known == refused to say (?)

Reply:
You're right, sorry misread this!